src/utils/runtime/src/logger.rs

```diff
@@ -492,6 +510,18 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
             .with_filter(reload_filter);
 
         layers.push(layer.boxed());
+
+        let reporter = OpenTelemetryReporter::new(
```

Comment:
May leave some comments showing that minitraces cannot be triggered by `enable_tracing` layer, but are controlled by foyer-related RPC.